code/game/gamemodes/objective_items.dm

```diff
@@ -33,7 +36,7 @@
 
 /datum/objective_item/steal/hoslaser
 	name = "the head of security's personal laser gun."
-	targetitem = /obj/item/gun/energy/e_gun/hos
+	targetitem = HOS_GUN
```

Comment:
objective_item datums don't support lists as targetitem right now, only typepaths. This is going to be an issue.